xbmc/platform/linux/input/LibInputKeyboard.cpp

```diff
@@ -151,6 +153,57 @@ constexpr auto xkbMap = make_map<xkb_keysym_t, XBMCKey>({
     // XBMCK_FASTFORWARD clashes with XBMCK_MEDIA_FASTFORWARD
 });
 
+/// Additional mappings directly from libinput (i.e. evdev) keycodes to Kodi keys
```

Comment:
explanation why this is necessary would be nice in the comment (i.e., they were forgotten in xkb and we don't want to wait for the merge request)

Reply:
Good point; added.